### PR TITLE
feat(Upstream): debug API

### PR DESCRIPTION
**Pull request by juzhiyuan**
Please answer these questions before submitting a pull request

- Why submit this pull request?
- [ ] Bug fix
- [ ] New feature provided
- [x] Improve performance

- Related issues

___
### Bugfix
- Description

- How to fix?

___
### New feature or improvement
- Describe the details and related test reports.

### CODE REVIEW

File: src/components/Upstream/UpstreamForm.tsx [hunk 1]
```diff
@@ -0,0 +1,602 @@
...
+            {fields.map((field, index) => (
+              <Form.Item
+                key={field.key}
+                label={
+                  index === 0 &&
+                  formatMessage({ id: 'upstream.step.healthy.checks.active.req_headers' })
+                }
+                wrapperCol={{ offset: index === 0 ? 0 : 3 }}
+              >
+                <Row style={{ marginBottom: '10px' }} gutter={16}>
```
Comment (LiteSun) [inline src/components/Upstream/UpstreamForm.tsx:342]: should use the same code style.`marginBottom: 10`

File: src/components/Upstream/UpstreamForm.tsx [hunk 2]
```diff
@@ -0,0 +1,602 @@
...
+                  <Col
+                    style={{
+                      marginLeft: -10,
+                      display: 'flex',
+                      alignItems: 'center',
+                    }}
+                  >
+                    {!readonly && fields.length > 1 && (
+                      <MinusCircleOutlined
+                        style={{ margin: '0 8px' }}
```
Comment (LiteSun) [inline src/components/Upstream/UpstreamForm.tsx:362]: same

File: src/components/Upstream/UpstreamForm.tsx [hunk 3]
```diff
@@ -0,0 +1,602 @@
...
+                required
+                key={field.key}
+                label={
+                  index === 0 &&
+                  formatMessage({ id: 'upstream.step.healthy.checks.passive.http_statuses' })
+                }
+                labelCol={{ span: index === 0 ? 3 : 0 }}
+                wrapperCol={{ offset: index === 0 ? 0 : 3 }}
+              >
+                <Row style={{ marginBottom: '10px' }}>
```
Comment (LiteSun) [inline src/components/Upstream/UpstreamForm.tsx:404]: ,

File: src/components/Upstream/UpstreamForm.tsx [hunk 4]
```diff
@@ -0,0 +1,602 @@
...
+                  <Col span={2}>
+                    <Form.Item style={{ marginBottom: 0 }} name={[field.name]}>
+                      <InputNumber disabled={readonly} />
+                    </Form.Item>
+                  </Col>
+                  <Col
+                    style={{
+                      marginLeft: -10,
```
Comment (LiteSun) [inline src/components/Upstream/UpstreamForm.tsx:412]: Extracts the same style as a variable.

**Comment by juzhiyuan** [comment]
@LiteSun Just ignore those checks above for this branch's API test cases are outdated, we just make sure the final PR which will be merged into master should pass all checks.

**Review by LiteSun — CHANGES_REQUESTED** [review]

**Pull request merged**